x/ecocredit/denom.go

```diff
@@ -97,12 +97,12 @@ var exponentPrefixMap = map[uint32]string{
 var validExponents string
 
 func init() {
-	var exponents = make([]uint32, len(exponentPrefixMap))
+	var exponents []uint32
```

Comment:
any reason for this change? imo its better to preallocate the memory like it did before

Reply:
Yes, if we use `make([]uint32, len(exponentPrefixMap))` the resultant array that we getting is `[0 0 0 0 0 0 0 0 0 0 0 0 1 2 3 6 9 12 15 18 21 24]` after `sort.Slice`

Reply:
right, we would need `make([]uint32, len(exponentPrefixMap), 0)`